### ISSUE TITLE
VS 2022 17.2 for Windows doesn't seem to reconize the vsix file

### ISSUE THREAD

**Issue by martyf5** [whitphx/vscode-emacs-mcx#1333]
5/27/2022 1:43:16 PM - Microsoft VSIX Installer
5/27/2022 1:43:16 PM - -------------------------------------------
5/27/2022 1:43:16 PM - vsixinstaller.exe version:
5/27/2022 1:43:16 PM - 17.2.2174
5/27/2022 1:43:16 PM - -------------------------------------------
5/27/2022 1:43:16 PM - Command line parameters:
5/27/2022 1:43:16 PM - C:\Program Files (x86)\Microsoft Visual Studio\Installer\resources\app\ServiceHub\Services\Microsoft.VisualStudio.Setup.Service\VSIXInstaller.exe,/a,C:\my-vs-extensions\tuttieee.emacs-mcx-0.44.0.vsix
5/27/2022 1:43:16 PM - -------------------------------------------
5/27/2022 1:43:16 PM - Microsoft VSIX Installer
5/27/2022 1:43:16 PM - -------------------------------------------
5/27/2022 1:43:16 PM - Initializing Install...
5/27/2022 1:43:16 PM - Extension Details...
5/27/2022 1:43:16 PM - 	Identifier         : emacs-mcx
5/27/2022 1:43:16 PM - 	Name               : Awesome Emacs Keymap
5/27/2022 1:43:16 PM - 	Author             : tuttieee
5/27/2022 1:43:16 PM - 	Version            : 0.44.0
5/27/2022 1:43:16 PM - 	Description        : Emacs Friendly Keymap with multi cursor support, improved mark-mode experience, clipboard and kill-ring integration, and lots of improvements and bug fixes.
5/27/2022 1:43:16 PM - 	Locale             : en-US
5/27/2022 1:43:16 PM - 	MoreInfoURL        : 
5/27/2022 1:43:16 PM - 	InstalledByMSI     : False
5/27/2022 1:43:16 PM - 	SupportedFrameworkVersionRange : [0.0,2147483647.2147483647]
5/27/2022 1:43:16 PM - 
5/27/2022 1:43:16 PM - 	SignatureState     : Unsigned
5/27/2022 1:43:16 PM - 	Supported Products : 
5/27/2022 1:43:16 PM - 		Microsoft.VisualStudio.Code
5/27/2022 1:43:16 PM - 			Version : 
5/27/2022 1:43:16 PM - 			ProductArchitecture : x86
5/27/2022 1:43:16 PM - 
5/27/2022 1:43:16 PM - 	References         : 
5/27/2022 1:43:16 PM - Signature Details...
5/27/2022 1:43:16 PM - 	Extension is not signed.
5/27/2022 1:43:16 PM - 
5/27/2022 1:43:16 PM - Searching for applicable products...
5/27/2022 1:43:16 PM - Found installed product - Global Location
5/27/2022 1:43:16 PM - Found installed product - Visual Studio Community 2022
5/27/2022 1:43:16 PM - VSIXInstaller.NoApplicableSKUsException: One or more extensions are for Visual Studio Code. Try installing them in Visual Studio Code.
   at VSIXInstaller.ExtensionService.GetInstallableDataImpl(IInstallableExtension extension, String extensionPackParentName, Boolean isRepairSupported, IStateData stateData, IEnumerable`1& skuData)
   at VSIXInstaller.ExtensionService.GetInstallableData(String vsixPath, String extensionPackParentName, Boolean isRepairSupported, IStateData stateData, IEnumerable`1& skuData)
   at VSIXInstaller.ExtensionPackService.IsExtensionPack(IStateData stateData, Boolean isRepairSupported)
   at VSIXInstaller.ExtensionPackService.ExpandExtensionPackToInstall(IStateData stateData, Boolean isRepairSupported)
   at VSIXInstaller.App.Initialize(Boolean isRepairSupported)
   at VSIXInstaller.App.Initialize()
   at System.Threading.Tasks.Task`1.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)

**Comment by whitphx** [comment]
This extension is for VSCode.
Read the error message.
> 5/27/2022 1:43:16 PM - VSIXInstaller.NoApplicableSKUsException: One or more extensions are for Visual Studio Code. Try installing them in Visual Studio Code.

**Comment by martyf5** [comment]
> This extension is for VSCode. Read the error message.
> 
> > 5/27/2022 1:43:16 PM - VSIXInstaller.NoApplicableSKUsException: One or more extensions are for Visual Studio Code. Try installing them in Visual Studio Code.

I did. I cannot find the extension on market place (using VS). So, I downloaded the vsix file to install manually, but am unable to find a way to install it manually. If I load the file using VS, it treats it as a binary file.  I saw VS was using VSIXINstaller to install extensions, and so, I tried that (resulting in the error I posted). While, I've written hundred of thousands of lines of code in my career, I'm new to VS, and "Try installing them in Visual Studio Code" doesn't mean anything to me. I went though a MS tutorial to create an extension to try to understand the issue, and that worked ok. So, perhaps I'll do a pull on the git repo for this extension, and try to load it that way.
